license_output/about-box_direct.html

```diff
@@ -0,0 +1,141 @@
+<div>
```

Comment:
not sure where you got this file from...

Reply:
the npm puller generates these files in the license_output folder.